src/components/EnterpriseApp/data/enterpriseCurationReducer.js

```diff
@@ -107,6 +172,19 @@ function enterpriseCurationReducer(state, action) {
         },
       };
     }
+    case SET_IS_NEW_ARCHIVED_COURSE: {
+      const isNewArchivedContent = getIsNewArchivedContentFromState(action.payload);
+      return {
+        ...state,
+        isNewArchivedContent,
+      };
+    }
+    case UPDATE_DISMISSED_ARCHIVED_COURSE: {
+      return {
+        ...state,
+        isNewArchivedContent: action.payload,
+      };
+    }
```

Comment:
I notice these are both triggered [here](https://github.com/openedx/frontend-app-admin-portal/pull/1160/files#diff-353876612b187c97b16c79ddd5eac4b45d88d8df513bb99096a327b23dd8aa06R40) upon update of `enterpriseHighlightedSets`, in a way it would seem they would overwrite each other.  Am I missing something here?

Reply:
Only the `setIsNewArchivedContent` is triggered in the file you linked. The `updateDismissedArchivedCourse` is triggered when the alert is dismissed in the `ContentHighlightArchivedAlert` component.